Frends.SFTP.UploadFiles/Frends.SFTP.UploadFiles/Definitions/FileTransporter.cs

```diff
@@ -57,7 +57,7 @@ public FileTransferResult Run(CancellationToken cancellationToken)
         try
```

Comment:
https://github.com/FrendsPlatform/Frends.SFTP/blob/600e6c339ed4749162ab943cb217a9ead8104206/Frends.SFTP.UploadFiles/Frends.SFTP.UploadFiles/Definitions/FileTransporter.cs#L296
This method can be marked as `static`.